### PR TITLE
Set Institution Section 103 Messages for IHL institutions only

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
All section 103 logic in `institution_builder.rb` should only apply to IHL institutions. Updated SQL in `InstitutionBuilder` to reflect this.

DB migration PR: https://github.com/department-of-veterans-affairs/gibct-data-service/pull/646

[ZenHub Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/8095)

## Testing done
Tested locally and QA reviewed

## Screenshots
N/A

## Acceptance criteria
- [x] The "Institution Summary" area of the "Additional Information" section of the institution profile page ONLY displays the "Protection against late VA payments:" indicator for institutions of type IHL.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

**Review by johnpaulashenfelter — CHANGES_REQUESTED** [review]
This migration is also in #646 so seems like it needs untangled?

File: app/models/institution_builder.rb [hunk 1]
```diff
@@ -641,26 +641,28 @@ def self.add_extension_campus_type(version_id)
...
   def self.add_sec103(version_id)
     str = <<-SQL
+      -- set default message for IHL institutions
+      UPDATE institutions SET section_103_message = 'No information available at this time'
+      FROM weams
+      WHERE weams.facility_code = institutions.facility_code
+        AND SUBSTRING(weams.facility_code, 1, 2) IN('11', '12', '13', '21', '22', '23', '31', '32', '33')
+        AND institutions.version_id = #{version_id};
+
       -- set message based on sec103s
```
Comment (johnpaulashenfelter) [inline app/models/institution_builder.rb:651]: Will the leave the data in the right state for existing records that already have the ''No information available at this time' message from the original default?

Comment (dmccurdybah) [reply app/models/institution_builder.rb:651]: Yes, it's fine. We're not concerned about the value of this field in institutions from previous version generation runs.

Comment (johnpaulashenfelter) [reply app/models/institution_builder.rb:651]: Fair enough.

**Review by johnpaulashenfelter — APPROVED** [review]
:shipit:

**Pull request merged**